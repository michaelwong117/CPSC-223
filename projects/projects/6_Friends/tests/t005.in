A,B
B,C
A,C
A,F
F,B
F,E
E,D
B,D
C,D
D,G
G,E
B,E
