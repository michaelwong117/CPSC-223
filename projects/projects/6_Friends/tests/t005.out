A, B have minimum path length 1
B, C have minimum path length 1
A, C have minimum path length 1
A, F have minimum path length 1
F, B have minimum path length 1
F, E have minimum path length 1
E, D have minimum path length 1
B, D have minimum path length 1
C, D have minimum path length 1
D, G have minimum path length 1
G, E have minimum path length 1
B, E have minimum path length 1
